sycl/test-e2e/ESIMD/aot_mixed.cpp

```diff
@@ -7,9 +7,9 @@
 //===----------------------------------------------------------------------===//
 // TODO: Enable on other GPUs once internal ticket is fixed
 // REQUIRES: ocloc && gpu-intel-gen12
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend=spir64_gen -Xs "-device tgllp" -o %t.sycl.out -DENABLE_SYCL=0 %s
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend=spir64_gen "-device tgllp" -o %t.sycl.out -DENABLE_SYCL=0 %s
```

Comment:
That is odd. In my notes that `-Xs` is required. Seems something has changed in driver, right?

Reply:
Nope, it's because we have `-Xsycl-target-backend` which is kind of similar to `-Xs` but lets you provide backend options for a specific target because we support compiling for multiple targets at once. `-Xs` will apply to all targets (I think), here it doesn't matter because there's only one.

If you want to use `-Xs` only, drop `-Xsycl-target-backend=spir64_gen "-device tgllp"` and use `-Xs "-device tgllp"

TLDR it's just the way the `RUN` command was originally written and I missed it.